include/misc/util.h

```diff
@@ -453,4 +453,42 @@ static inline s64_t arithmetic_shift_right(s64_t value, u8_t shift)
 	_for_10, _for_9, _for_8, _for_7, _for_6, _for_5, \
 	_for_4, _for_3, _for_2, _for_1, _for_0)(x, ##__VA_ARGS__)
 
+/* FOR_EACH_FIXED_ARG is used for calling the same function
+ * With one fixed argument and changing 2nd argument.
+ */
+
+#define _rep_0(_fn, f, ...)
```

Comment:
`_rep_0(_fn, f)` ?

Reply:
start these macros with underscore will violate rules 21.5 and 21.6, though this will be addressed in other fix, would be nice we introduce it with the right namespace like is described in #9882